src/flamegraph.js

```diff
@@ -72,20 +105,19 @@ export default function () {
     return getName(d) + ' (' + format('.3f')(100 * (d.x1 - d.x0), 3) + '%, ' + getValue(d) + ' samples)'
   }
 
-  var tip = d3Tip()
-    .direction('s')
-    .offset([8, 0])
-    .attr('class', 'd3-flame-graph-tip')
-    .html(function (d) { return labelHandler(d) })
-
-  var svg
-
   function setSearchDetails () {
     detailsElement.innerHTML = searchSum + ' of ' + totalValue + ' samples ( ' + format('.3f')(100 * (searchSum / totalValue), 3) + '%)'
   }
 
+  var classMapper = function (d, base) {
+    let classes = base
+    if (d.fade) classes += ' stem'
+    if (d.highlight) classes += ' highlight'
+    return classes
+  }
+
   var colorMapper = function (d) {
-    return d.highlight ? '#E600E6' : colorHash(getName(d), getLibtype(d), getDelta(d))
```

Comment:
Setting a class and using CSS to set the color of the highlights. Is there a reason why the CSS only sets the border color and not the background color?

Reply:
I just find it a bit hard to visualize the highlighted frames with just a different border color.